{"name": "test_add_user_invalid_data[username11-None-user username11 does not exist-user_password]", "status": "passed", "steps": [{"name": "add user {'user_name': 'username11', 'user_password': None}", "status": "passed", "start": 1679691556867, "stop": 1679691556884}, {"name": "getting the user username11", "status": "passed", "start": 1679691556884, "stop": 1679691556900}], "attachments": [{"name": "log", "source": "b08c7a33-3ae4-4b8c-9c25-7344652e50e4-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "2ba2f0f8-76ba-49a3-be12-c7686615f534-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'username11'"}, {"name": "password", "value": "None"}, {"name": "msg", "value": "'user username11 does not exist'"}, {"name": "param", "value": "'user_password'"}], "start": 1679691556867, "stop": 1679691556901, "uuid": "e1f6bd6f-a187-40e0-a817-081cd6232179", "historyId": "97834a333815666047c6c2fcadda043d", "testCaseId": "c844ad0fbf3a81975d6b3ec8e1625f1e", "fullName": "tests.test_users.TestUsers#test_add_user_invalid_data", "labels": [{"name": "epic", "value": "users api"}, {"name": "story", "value": "a client need to insert username and password"}, {"name": "severity", "value": 2}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users"}, {"name": "subSuite", "value": "TestUsers"}, {"name": "host", "value": "DESKTOP-9JHRV77"}, {"name": "thread", "value": "8036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users"}]}